### ISSUE TITLE
It is not clear which date is being displayed on the toolbar

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1575]
The document byline viewlet was pretty clear on stating which date was displaying; on the new toolbar that is not the case:

![selection_073](https://cloud.githubusercontent.com/assets/341393/15050622/33f12b76-12cb-11e6-8e2c-f1b2bc28d719.png)

**Comment by djay** [comment]
Perhaps should be a tooltip or more information in the dropdown. @plone/ux-team ideas?

**Comment by tkimnguyen** [comment]
It seems to me the "last modified date" is most useful, if we have to choose, given the small amount of space in the toolbar.

**Comment by jensens** [comment]
Closing as outdated (10 years old). About Plone 5 toolbar date display. Toolbar has been reworked in Plone 6 Classic UI, and Volto uses a completely different UI.
If you think this issue is still relevant please reopen it. Thank you for the original report!
